eidpauth/src/test/java/uk/ac/ucl/eidp/auth/model/RoleeNgTest.java

```diff
@@ -0,0 +1,127 @@
+package uk.ac.ucl.eidp.auth.model;
+
+import static org.testng.Assert.*;
```

Comment:
\[Checkstyle\] WARNING: Using the '\.\*' form of import should be avoided \- org\.testng\.Assert\.\*\.